builds/service-onboarding-build-pack/Jenkinsfile

```diff
@@ -133,6 +135,11 @@ node  {
 				service_template = "${serviceType}-template-go"
 			}
 		}
+		} else {			
```

Comment:
Looks like a merge issue again - 
- Two events - one  with Success and another with Failure status in the code block.
- `else` block looks misplaced. Please check this again.

Reply:
Addressed this.